### ISSUE TITLE
[Feature Request] Add 'Mark as completed'

### ISSUE THREAD

**Issue by DrZingo** [sabnzbd/sabnzbd#1174]
I'd like a feature where you can mark a job as completed. Sometimes I manually fix incompleted jobs, and thus when I delete the job, the dublicate check won't kick in.
Maybe in the 'job pulldown tab' would be a good place?

**Comment by Safihre** [comment]
Interesting idea.
And when you do that, should it also remove the incomplete folder (if it's still left)? 
In general it would be nice addition to be able to Retry certain steps of the post processing.

**Comment by DrZingo** [comment]
Yes, a removal of the incomplete folder works for me. If I remember correctly that is the default behavior when a job is completed.

**Comment by Safihre** [comment]
<img width="269" height="116" alt="Image" src="https://github.com/user-attachments/assets/9781bbdd-cd0c-4da8-bd28-e479b88b3ec6" />

<img width="322" height="138" alt="Image" src="https://github.com/user-attachments/assets/8f9167ff-c2a7-4895-81b0-8048a76f7787" />
